docs/modules/ROOT/pages/how-to-guides/managing-applications/proc-managing_applications.adoc

```diff
@@ -0,0 +1,91 @@
+= Managing applications in secure development environment
+
+This feature enhances the handling of the sensitive content within the development environment. You can efficiently address security fixes while maintaining confidentiality and facilitating the recreation of applications.
```

Comment:
"Development Environment" is a specific concept within the context of their application - that's where their app's latest build is automatically deployed. This feature is more about the application itself. I would consider something like this:

```suggestion
This feature enables copying a selected application to a different workspace, which can have a different set of collaborators defined. This can be beneficial for working on secure fixes, without broadcasting those changes to all team members in the original application. 
```
If you want, you could also mention that since workspaces can't be renamed, it could also be helpful for moving an app to a new workspace.